_data/entertainment.yml

```diff
@@ -68,36 +75,42 @@ websites:
     - name: Plex
       url: https://www.plex.tv
       twitter: plexapp
+      facebook: TVPlex
       img: plex.png
       tfa: No
 
     - name: Sling TV
       url: https://www.sling.com/
       twitter: slinganswers
+      facebook: sling
       img: sling.png
       tfa: No
 
     - name: SoundCloud
       url: https://soundcloud.com/
       twitter: SCsupport
+      facebook: SoundCloud
       img: soundcloud.png
       tfa: No
 
     - name: Spotify
       url: https://www.spotify.com/
       twitter: spotifycares
+      facebook: spotifyusa
```

Comment:
Same comment as with Netflix. I think [Spotify](https://www.facebook.com/Spotify) should be used instead.